releasenotes-builder/src/main/java/com/github/checkstyle/CliOptions.java

```diff
@@ -558,6 +592,18 @@ public Builder setMlistTemplate(String mlistTemp) {
             return this;
         }
 
+        /**
+         * Specify GitHub Post template.
+         *
+         * @param GitHubPageTemp GitHub post template
+         * @return Builder Object
+         * @noinspection ReturnOfInnerClass
+         */
+        public Builder setGitHubTemplate(String GitHubPageTemp) {
```

Comment:
violation in Teamcity
`src/main/java/com/github/checkstyle
CliOptions.java (1)
[602: setGitHubTemplate() Parameter name GitHubPageTemp doesn't match regex '[a-z][A-Za-z\d]*'

Reply:
Changes Done